NavigationSystem/ASPire.mk

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+#
+# ASPire Makefile
+#
+###############################################################################
+
+
+###############################################################################
+# Files
+###############################################################################
+
+MAIN = main_ASPire.cpp
+OBJECTS = $(addprefix $(BUILD_DIR)/, $(SRC:.cpp=.o))
+
+HARDWARE_SERVICES_SRC = Hardwares/i2ccontroller/I2CController.cpp \
+							   Hardwares/CAN_Services/CANPGNReceiver.cpp Hardwares/CAN_Services/CANService.cpp \
+							   Hardwares/CAN_Services/mcp2515.cpp Hardwares/CAN_Services/MsgFunctions.cpp \
+							   Hardwares/CAN_Services/CANFrameReceiver.cpp
+
+HARDWARE_NODES_SRC   = Hardwares/HMC6343Node.cpp Hardwares/GPSDNode.cpp Hardwares/ActuatorNodeASPire.cpp \
+                            Hardwares/CANFeedbackReceiver.cpp Hardwares/CANWindsensorNode.cpp
+
+CORE_SRC             = 	Navigation/WaypointMgrNode.cpp LowLevelControllers/LowLevelControllerNodeASPire.cpp \
+										WorldState/StateEstimationNode.cpp WorldState/WindStateNode.cpp WorldState/VesselStateNode.cpp \
+										$(MESSAGE_BUS_SRC) $(NETWORK_SRC) $(SYSTEM_SERVICES_SRC) $(MATH_SRC)
+ifeq ($(USE_LNM),1)
+SRC 											= $(MAIN_LNM_SRC) $(CORE_SRC) $(LNM_SRC)
```

Comment:
One main ASPire and one main Janet instead of a main LNM and a main line-follow.